### ISSUE TITLE
(DESIGN/UX/ACCESSIBILITY) Map Pin Improvements

### ISSUE THREAD

**Issue by KirstyScott** [visitscotland/vs-component-library#98]
**Task**

Display the name of the product/place name (alt text) over the pin when the user hovers over or selects a pin.

**Acceptance Criteria (Desktop)**
- Main Map displays the place name or polygon region name when the user hovers over the pin 
- Regional map displays the iCentre location name or Place name when the user hovers over the pin 
- Destinations map displays product name or event location when the user hovers over the pin 
- Outlander map to display place name when the user hovers over the pin 
- When you select an item from the menu it should be clearer which pin is related to it so the user can easily identify the location

**Tasks** 

- [ ] Design and UX to look at colour palette in the first instance 
- [ ] Review and update hover state of pin.  It is currently not obvious enough
- [ ] Review this from an Accessibility stand point 
- [ ] Decide if event location should display 
- [ ] Decide what results should display in relation to level of zoom 

**Considerations**
- When a user with a screen reader focuses on the map pins, the locations should be read out - currently, when the screen reader tabs through the map, the map pins read out "button."
- For events, we need to check whether the event location will display as there could be multiple events on for one location (conversation relating to this can be found in #106 )
- Results in the menu can display items that are not within the current view of the map. Should the results only display if current level of zoom allows it? Or should the zoom level of the map change to display the location if the pin is selected? This behaviour would probably be detract from the user experience if it behaved the same on hover as the map would be frequently moving. (needs own ticket)
